kubernetes/namespaces/web/site/deployment.yaml

```diff
@@ -13,6 +13,19 @@ spec:
       labels:
         app: site
     spec:
+      initContainers:
+        - name: migrations
+          image: ghcr.io/python-discord/site:latest
+          imagePullPolicy: Always
+          command:
+            - python
```

Comment:
Because we're setting the entrypoint to `poetry run`, we can omit it here, right?

Reply:
Hmm, it is omitted here, this would be the equivalent of `[poetry run] python manage.py migrate` right?